course_gather/settings/dev_settings.py

```diff
@@ -14,6 +14,10 @@
     'debug_toolbar.middleware.DebugToolbarMiddleware',
 )
 
+REST_FRAMEWORK['DEFAULT_RENDERER_CLASSES'] += (  # noqa: F405
```

Comment:
Do we want to enable the docs for prod as well (or just put in base settings)?

Reply:
This specific line change is for the Django browsable API, not the API docs.
API docs are enabled in prod, via the `installed_apps` and in `urls.py`

Reply:
Took a look at the docs and it makes more sense now, my bad.